ch05construction/03comments.ipynb

```diff
@@ -116,25 +126,22 @@
    "outputs": [],
    "source": [
     "for agent in agents:\n",
-    "  agent.turn()\n",
-    "  agent.move()\n",
+    "    agent.turn()\n",
+    "    agent.move()\n",
     "\n",
-    "class Agent(object):\n",
-    "   def turn(self):\n",
-    "        self.direction+=self.angular_velocity;\n",
-    "   def move(self):\n",
-    "       self.x+=Agent.step_length*sin(self.direction)\n",
-    "       self.y+=Agent.step_length*cos(self.direction)"
+    "class Agent:\n",
+    "    def turn(self):\n",
+    "         self.direction += self.angular_velocity;\n",
+    "    def move(self):\n",
+    "        self.x += Agent.step_length * sin(self.direction)\n",
+    "        self.y += Agent.step_length * cos(self.direction)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "\n",
-    "\n",
-    "\n",
-    "is probably better.\n"
+    "This is probably better. We are using the name of the functions (_i.e._, `turn`, `move`) instead of comments. Therefore, we've got _self-documented_ code.\n"
```

Comment:
```suggestion
    "This is probably better. We are using the name of the functions (_i.e._, `turn`, `move`) instead of comments. Therefore, we've got _self-documenting_ code.\n"
```